### PR TITLE
✏ Add lsb-image and joke codecs

### PR DESCRIPTION
**Pull request by thatgaypigeon**
Font (Chakra Petch) is available via the [Open Font License](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL).

### CODE REVIEW

File: src/pydis_jam23/codecs/notlsb.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+"""Encode image and decode functions for the joke codec."""
```
Comment (falkanson) [inline src/pydis_jam23/codecs/notlsb.py]: Could you explain what the codec actually does?

**Review by falkanson — CHANGES_REQUESTED** [review]
Looks good and pretty clean.

But you should probably comment a bit more on the functionality. and change cli_flag and cli_help to something more descriptive.

**Comment by thatgaypigeon** [comment]
Yeah, I didn't really bother with too much documentation as I thought that'd be something we do after the deadline. I was just focused on functionality. If I can manage it, I'll add another codec if possible then update the docs etc. after that :)

**Review by Artemis21 — APPROVED** [review]
Can't pretend this is a thorough review but most of it works

**Review by falkanson — APPROVED** [review]

**Pull request merged**